wordpress-mobile/WordPress-iOS
# [Stats Refresh] add date bar to Periods table.

Fixes n/a

This adds the date changer bar to the top of the Periods table. 

What it does:
- Displays the day, week, month, year for the current date.
- The week is the week containing the date, Monday - Sunday.
- The date bar remains visible when scrolling.

What it doesn't do (yet):
- The back and forward buttons aren't hooked up.
- Which means the date can't actually be changed.

To test:
- Go to Stats Period, and select the different periods.
- Verify the correct date period is displayed.

<img width="350" alt="days" src="https://user-images.githubusercontent.com/1816888/55032427-75d50080-4fd6-11e9-9325-63189c621d12.png">

<img width="350" alt="weeks" src="https://user-images.githubusercontent.com/1816888/55032429-79688780-4fd6-11e9-96c2-33ffe6b56e7a.png">

<img width="350" alt="months" src="https://user-images.githubusercontent.com/1816888/55032443-7cfc0e80-4fd6-11e9-9b7c-9a8314611b5e.png">

<img width="350" alt="years" src="https://user-images.githubusercontent.com/1816888/55032448-808f9580-4fd6-11e9-908d-cb205a1d1e11.png">
